conans/client/build/meson.py

```diff
@@ -126,3 +171,29 @@ def build(self, args=None, build_dir=None, targets=None):
         command = "ninja %s" % arg_list
         command = self._append_vs_if_needed(command)
         self._conanfile.run(command)
+
+    def install(self, args=None, build_dir=None):
+        if not self._conanfile.should_install:
+            return
+        mkdir(self._conanfile.package_folder)
+        if not self.options.get('prefix'):
+            raise ConanException("'prefix' not defined for 'meson.install()'\n"
+                                 "Make sure 'package_folder' is defined")
+        self.build(args=args, build_dir=build_dir, targets=["install"])
+
+    def test(self, args=None, build_dir=None, targets=None):
+        if not self._conanfile.should_test:
+            return
+        if not targets:
+            targets = ["test"]
+        self.build(args=args, build_dir=build_dir, targets=targets)
+
+    @staticmethod
+    def get_version():
```

Comment:
This might require some test, right now we don't have a wait to check/exclude by default tests like this one, but it is planned. Lets leave it, I guess you have tested it manually.

Reply:
"we don't have a wait to check/exclude by default tests" => "way to exclude".

I meant, that by default, the tests should pass for contributors, without requiring them to install Meson. And testing Meson would be something that could be opt-in.